### DISCUSSION TITLE
How to implement wingloss with mxnet?

### DISCUSSION THREAD

**Discussion by guoguoguilai** [apache/incubator-mxnet#14932] [Q&A]
How to implement wingloss with mxnet?

[a comment by an automated account is not shown]

**Comment by vrakesh** [comment]
@guoguoguilai Thank you for question, have you looked into creating custom operators??

https://mxnet.incubator.apache.org/versions/master/faq/new_op.html


Hope this helps

@mxnet-label-bot  add [Question]
